docs/pages/User/GettingStarted/uninstall.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Uninstall Liqo
+weight: 4
+---
+
+## Uninstall steps
+
+This procedure uninstalls Liqo on your cluster.
+
+```bash
+curl https://raw.githubusercontent.com/LiqoTech/liqo/master/install.sh | bash -s -- --uninstall
+```
+
+### Delete CRDs
```

Comment:
Can we replace "CRD" with something simpler? Not sure who will test Liqo has full understanding of what CRD means.

Reply:
What about something like "Delete all stored resources"?

Reply:
Maybe something like "Purging data"?

Reply:
"Purge LIQO data" looks definitely simpler.
However, wondering why this has to be chosen by the user. I assume, by default, that if I want to uninstall LIQO, data should be removed. I don't see reason for keeping Liqo data alive.